charts/spire/charts/spire-server/templates/tests/test-connection.yaml

```diff
@@ -16,4 +16,12 @@ spec:
       args: ['-zvw3', '{{ include "spire-server.fullname" . }}', '{{ .Values.service.port }}']
       securityContext:
         {{- toYaml .Values.securityContext | nindent 8 }}
+    {{- if eq (Values.federation.bundleEndpoint.enabled | toString) "true" }}
+    - name: wget-federation-bundle-endpoint
+      image: busybox
+      command: ['wget']
+      args: ['--no-check-certificate', '-O', '/dev/null', 'https://{{ include "spire-server.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.federation.bundleEndpoint.port }}']
```

Comment:
it turns out `--no-check-certificate` isn't required because busybox wget doesn't implement certificate verification. Should I leave it here anyway in case this gets implemented or the image is switched out?

Reply:
Yeah, I'd leave it. Its a nice breadcrumb to know about I think.